lib/apiservers/engine/backends/archive.go

```diff
@@ -287,9 +273,11 @@ func NewArchiveStreamReaderMap(mounts []types.MountPoint) *ArchiveStreamReaderMa
 		//
 		// Neither the volume nor the storage portlayer knows about /mnt/A.  The persona must tell
 		// the portlayer to rebase all files from this volume to the /mnt/A/ in the final tar stream.
-		if ar.mountPoint.Destination != "/" {
-			ar.filterSpec.RebasePath = ar.mountPoint.Destination
-		}
+
+		// hickeng: the current logic is rebase, then strip for export - more efficient to collapse
+		// any reducdency here than apply to every tar entry when packing but skipping for now.
+		ar.filterSpec.RebasePath = ar.mountPoint.Destination
```

Comment:
This will not work for every case, but should be addressed in the FilterSpec generation PR.